job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/SlocCountContext.groovy

```diff
@@ -0,0 +1,49 @@
+package javaposse.jobdsl.dsl.helpers.publisher
+
+import javaposse.jobdsl.dsl.Context
+
+/**
+ * DSL for https://wiki.jenkins-ci.org/display/JENKINS/SLOCCount+Plugin
+ */
+class SlocCountContext implements Context {
+    String pattern = ''
```

Comment:
Only initialize the field if deviating from the default value. Handle null values when generating the XML. Users can try to set the value to null, so that must be handled anyway.